### ISSUE TITLE
dataset split

### ISSUE THREAD

**Issue by feililucky** [FengheTan9/Multi-Level-Global-Context-Cross-Consistency#13]
Hello, I saw in your paper that we utilize all cases and randomly split the BUSI dataset into 70-30 ratios three times (i.e., 526 samples for training and 254 samples for validating) to ensure fair comparison., but I based on In your split.py code, I found that the final training set is 546 and the test set is 234. Which one should I use as the standard? Thank you
